app/controller/member.controller.js

```diff
@@ -42,6 +42,7 @@ exports.findOne = async (req, res) => {
             nickname: result.nickname,
             email: result.email,
             age: result.age,
+            profile_img_url: result.profile_img_url,
```

Comment:
Codacy has a fix for the issue: [Replace `············` with `↹↹↹↹`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9683172)
```suggestion
				profile_img_url: result.profile_img_url,
```